modern/src/reports/ReplayPage.js

```diff
@@ -41,58 +62,138 @@ const TimeLabel = ({ children, open, value }) => (
 
 const ReplayPage = () => {
   const classes = useStyles();
+  const history = useHistory();
 
-  const [expanded, setExpanded] = useState(true);
   const [positions, setPositions] = useState([]);
   const [index, setIndex] = useState(0);
+  const [selectedDeviceId, setSelectedDeviceId] = useState();
+
+  const deviceName = useSelector((state) => {
+    if (selectedDeviceId) {
+      const device = state.devices.items[selectedDeviceId] || null;
+      if (device) {
+        return device.name;
+      }
+    }
+    return null;
+  });
+
+  const [playbackPerMin, setPlaybackPerMin] = useState('');
+  const [autoPlay, setAutoPlay] = useState(false);
+  const [isPlaying, setIsPlaying] = useState(false);
```

Comment:
It looks like you replaced `expanded` with this, but I don't think it's conceptually correct. You can potentially have history playing and configuration expanded.